parser/types/field_type.go

```diff
@@ -58,7 +59,12 @@ func NewFieldType(tp byte) *FieldType {
 		decimal: UnspecifiedLength,
 	}
 }
-
+func (ft *FieldType) IsDecimalValid() bool {
```

Comment:
We already have a similar function named `isValidTiFlashDecimalType`, I think we can reuse it?

Reply:
delete that, use the new is better.